portal/docs/overview/overview.md

```diff
@@ -26,5 +26,5 @@ Kiva piloted the API with Milaap, one of our Field Partners working in India, wi
 ## How do I start using Kiva’s API?
 Kiva’s API is set up as an endpoint that accepts data in a specific format using JSON. To begin using our API, the Field Partner needs to set up their MIS to send information to Kiva’s endpoint in this format.
 
-If you are interested in integrating in 2021, please complete this [form](https://kiva.tfaforms.net/107 "form").
+If you are interested in integrating with the API, please complete this [form](https://kiva.tfaforms.net/107 "form").
```

Comment:
does this change need to be reflected in the ES and FR pages as well?

Reply:
No, those pages already have the proper text:

> Si vous êtes intéressé à rejoindre le projet API, veuillez remplir ce [formulaire](https://kiva.tfaforms.net/119).
translates to
> If you are interested in joining the API project, please fill out this form.

and
> Si está interesado en integrarse, por favor, rellene este [formulario](https://kiva.tfaforms.net/111).
translates to
> If you are interested in joining us, please fill in this form.

Reply:
or at least.. they don't reference 2021, hah. I'm not tooo interested in scrutinizing the translations for this suite of changes; I'm hoping to make a quick pass with the goal of making them legible and populated with the proper links (even if the content in those links are not internationalized).

Reply:
While we shouldn't need to scrutinize the translations, they need to be consistent and we need to be mindful of that as usability is affected for those users when the consistency isn't there.  Since the language files do not explicitly call out 2021 we are probably good.

Reply:
yup, and I did check for 2021 in the other translations when I made the change.